compiler/hie-core/README.md

```diff
@@ -61,10 +61,9 @@ Now openning a `.hs` file should work with `hie-core`.
 
 ### Using with Emacs
 
-The frst step is to install required Emacs packages. If you don't already have [Melpa](https://melpa.org/#/) package installation configured in your `.emacs`, put this stanza at the top.
-
+If you don't already have [Melpa](https://melpa.org/#/) package installation configured in your `.emacs`, put this stanza at the top.
 ```elisp
```

Comment:
Maybe we should just link to https://melpa.org/#/getting-started instead of copying the melpa installation instructions here?